src/components/Spinner/Spinner.js

```diff
@@ -52,8 +53,18 @@ fabric.Spinner = function(target) {
     //private methods
 
     function _init() {
+        //for backwards compatibility
+        if (_target.className.indexOf("ms-Spinner") === -1) {
+            spinner = document.createElement("div");
+            spinner.className = "ms-Spinner";
+            _target.appendChild(spinner);
+        } else {
+            spinner = _target;
+        }
+
         offsetSize = eightSize;
         numCircles = 8;
+        parentSize = spinner.className.indexOf("large") > -1 ? 28 : 20;
```

Comment:
Do we think 28px and 20px is enough of a difference? Maybe we can find examples of this in product to see what sizes are most common, it just seems a little small to me.